core/mapper/exprmapper/expression/expr/expr.go

```diff
@@ -51,9 +51,9 @@ func (e *Expression) IsNil() bool {
 }
 
 type TernaryExpressio struct {
```

Comment:
TernaryExpressio should be TernaryExpression

Reply:
Thanks @fm-tibco,  Fixed the typo